src/main/xerus/monstercat/api/Player.kt

```diff
@@ -231,9 +239,12 @@ object Player: FadingHBox(true, targetHeight = 25) {
 	fun updateCover(coverUrl: String?) {
 		logger.debug("Updating cover: $coverUrl")
 		this.coverUrl = coverUrl
-		checkFx {
-			backgroundCover.value = coverUrl?.let {
-				Background(BackgroundImage(Covers.getCoverImage(coverUrl), BackgroundRepeat.NO_REPEAT, BackgroundRepeat.NO_REPEAT, BackgroundPosition.CENTER, BackgroundSize(100.0, 100.0, true, true, true, true)))
+		GlobalScope.launch {
```

Comment:
I think we can avoid this change if we simply enable backgroundLoading for the Image created in `createImage`. SHould be a constructor parameter.

Reply:
nevermind, doesn't work for InputStreams